README.md: reorg the content in README.md (#53)
pingcap/community: README.md

# TiDB Community

Welcome to the TiDB Community! The main objective is to help members of the
TiDB community who share similar interests to learn from and collaborate with
each other.

Your journey of becoming a contributor and committer starts from here:
improving docs, improving code, giving talks, organizing meetups, etc.

## Architecture

![TiDB Community Architecture](./architecture.svg)

## TiDB User Group

> The TiDB User Groups are groups for facilitating communication and discovery of

Reviewer: ```suggestion
The TiDB User Groups (TUGs) are groups for facilitating communication and discovery of
```

information related to topics that have long term relevance to large groups of
TiDB users.

See [docs](https://pingcap.com/docs/) or
[docs-cn](https://pingcap.com/docs-cn/) for user documents. Also you can get
help in [AskTUG](https://asktug.com/) if you met any problem.

## TiDB Developer Group

> ### Communicating

Reviewer: ```suggestion
### Communication
```

The [communication](./communicating.md) page lists communication channels like
chat, social medias, etc.

> For more specific topics, try to join

Reviewer: ```suggestion
For more specific topics, join
```

[https://pingcap.com/tidbslack](https://pingcap.com/tidbslack) and discuss with others.

### Governance

TiDB has the following types of groups that are officially supported:

* **Special Interest Groups (SIGs)** are persistent open groups that focus on a

>   part of the TiDB. SIGs must have open and transparent proceedings. Anyone is

Reviewer: ```suggestion
  module of TiDB. SIGs must have open and transparent proceedings. Anyone is
```

  welcome to participate and contribute provided they follow the TiDB [Code of
  Conduct](./CODE_OF_CONDUCT.md). The purpose of a SIG is to own and develop a

>   part of TiDB.

Reviewer: ```suggestion
module of TiDB.
```

* **Working Groups (WGs)** are temporary groups that are formed to address
  issues that cross SIG boundaries. Working groups do not own any code or other
  long term artifacts. Working groups can report back and act through involved
  SIGs.

### How to contribute

Contributions are welcomed and greatly appreciated. See
[CONTRIBUTING.md](./CONTRIBUTING.md)
for details on submitting patches and the contribution workflow.

You can start by finding an existing issue with the
[help-wanted](https://github.com/pingcap/tidb/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22)

> label in the tidb repo. These issues are well suited for new contributors. BTW,

Reviewer: ```suggestion
label in the tidb repository. These issues are well suited for new contributors. BTW,
```

there's always a need for more test coverage. You can also start with an area
that has [low coverage](https://codecov.io/gh/pingcap/tidb).

You can also join one of the [active working groups](./working-groups) to

> participate in a community activity. Start attending working group meetings and

Reviewer: ```suggestion
participate in a community activity. Start with attending working group meetings or
```

Reviewer: > Start attending working group meetings and

This sentence feels disconnected from the context. So I assume it's a suggestion on how to join working groups.

> join the slack channel.

Reviewer: ```suggestion
joining the slack channel.
```

Some useful documents:

* [Github Workflow](./contributors/workflow.md)
* [SRE-BOT Command Help](./contributors/command-help.md)
* [Become a Committer](./become-a-committer.md)

## Learning Resources

* [How we build TiDB](https://www.pingcap.com/blog/2016-10-17-how-we-build-tidb/)
* [A Brief Introduction of TiDB](https://www.pingcap.com/blog/2017-05-23-perconalive17/)
* [https://pingcap.com/docs](https://pingcap.com/docs)

> * [TiDB 源码阅读系列文章](https://pingcap.com/blog-cn/#TiDB-%E6%BA%90%E7%A0%81%E9%98%85%E8%AF%BB), in chinese

Reviewer: Suggest removing the Chinese links. Maybe some other articles in English, like TiDB Internal I II III? ...

Author: Have we translated the TiDB Internal articles?

Reviewer: We have. 
https://pingcap.com/blog/2017-07-11-tidbinternal1/
https://pingcap.com/blog/2017-07-11-tidbinternal2/
https://pingcap.com/blog/2017-07-11-tidbinternal3/

Author: They're focused on the tidb architecture, while these Chinese articles are focused on the source code reading, they're different for the developers. I think both of them should be added to the learning resources.

Author: BTW, we're going to use a big section to collect all the learning resources. It will be refactored in the future.

Reviewer: OK

* [TiKV 源码解析系列文章](https://pingcap.com/blog-cn/#TiKV-%E6%BA%90%E7%A0%81%E8%A7%A3%E6%9E%90), in chinese

## TiDB Roadmap

See [TiDB Roadmap](https://pingcap.com/docs/v2.1/roadmap/#tidb-roadmap).

## Tutorial Content

> If you're interested in writing a tutorial for TiDB, please contact us:

Reviewer: This section feels strange - we don't offer anything tutorial, but instead, we ask for contribution....

Author: Yeah, I think we should remove it from this file.

[community@pingcap.com](mailto:community@pingcap.com)

## License

TiDB Community is under the Apache 2.0 license. See the
[LICENSE](./LICENSE.md) file for details.

## Acknowledgements

Thank you to the [Kubernetes](https://github.com/kubernetes/community),
[Apache](http://activemq.apache.org/becoming-a-committer.html) and
[Docker](https://github.com/docker/community) community pages for providing us

> with inspiration.

Reviewer: ```suggestion
with inspirations.
```

# TiDB Community

Welcome to the TiDB Community! The main objective is to help members of the
TiDB community who share similar interests to learn from and collaborate with
each other.

Your journey of becoming a contributor and committer starts from here:
improving docs, improving code, giving talks, organizing meetups, etc.

> 

Reviewer 2: I recommend keeping this part because this can give the audience a direction of how to engage when they first come.
